src/main/java/core/basesyntax/service/impl/ActivityServiceImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.enumeration.Operation;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ActivityService;
+import core.basesyntax.strategy.FruitHandler;
+import core.basesyntax.strategy.impl.BalanceFruitHandlerImpl;
+import core.basesyntax.strategy.impl.PurchaseFruitHandlerImpl;
+import core.basesyntax.strategy.impl.ReturnFruitHandlerImpl;
+import core.basesyntax.strategy.impl.SupplyFruitHandlerImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ActivityServiceImpl implements ActivityService {
+    private final Map<Operation, FruitHandler> fruitStrategyMap = new HashMap<>() {{
+            put(Operation.BALANCE, new BalanceFruitHandlerImpl());
+            put(Operation.SUPPLY, new SupplyFruitHandlerImpl());
+            put(Operation.PURCHASE, new PurchaseFruitHandlerImpl());
+            put(Operation.RETURN, new ReturnFruitHandlerImpl());
+            }};
```

Comment:
This is the core functionality of the current task - strategy pattern implementation.
Please move  code that creates a map to separate class in `strategy` package.
Create in this new class method `FruitHandler get(FruitTransaction.Operation operation)` that will return specific handler by the operation.